src/components/Books/Book/Book.js

```diff
@@ -34,15 +38,76 @@ const Book = () => {
     retrieveFromApi();
   }, []);
 
+  const percentage = 66;
+
   return books.map((bookObj) => (
-    <li key={bookObj.id}>
-      {`${bookObj.title} -- ${bookObj.author} `}
-      <button
-        type="button"
-        onClick={() => removeBookHandler(bookObj, bookObj.id)}
-      >
-        remove
-      </button>
+    <li
+      key={bookObj.id}
+      className="py-8 pl-7 mb-4 bg-white border rounded-md flex  items-center"
+    >
+      <div className="border-r-2 h-full w-3/4 flex justify-between items-center">
+        <div>
+          <h3 className="font-bold text-2xl">
+            {bookObj.title}
+          </h3>
+          <p className="text-blue-500 font-light">
+            {bookObj.author}
+          </p>
+          <ul className="flex items-center space-x-5 mt-5">
+            <li>
+              <button
+                className="text-blue-500 font-light"
+                type="button"
+              >
+                Comments
+              </button>
+            </li>
+            <li>
+              <button
+                className="text-blue-500 font-light border-l border-r px-5"
+                type="button"
+                onClick={() => removeBookHandler(
+								    bookObj,
+								    bookObj.id,
+								  )}
+              >
+                Remove
+              </button>
+            </li>
+            <li>
+              <button
+                className="text-blue-500 font-light"
+                type="button"
+              >
+                Edit
+              </button>
+            </li>
+          </ul>
+        </div>
+        <div
+          className="mx-9 lg:mx-24"
+          style={{ width: 100, height: 100 }}
+        >
+          <CircularProgressbar
+            value={percentage}
+            text={`${percentage}%`}
+          />
+        </div>
+      </div>
+      <div className="h-full w-1/4 mx-9 lg:ml-24 lg:-mr-6">
+        <p className="text-gray-400 font-extralight text-sm">
+          CURRENT CHAPTER
+        </p>
+        <p className="text-gray-600 text-sm mt-1 mb-4">
+          CHAPTER 12
+        </p>
+        <button
+          className="font-extralight min-w-max px-3 md:w-4/6 lg:w-3/6 h-8 border border-blue-600 rounded-md bg-blue-600 text-white text-sm cursor-pointer hover:bg-blue-500"
+          type="button"
+        >
+          UPDATE PROGRESS
+        </button>
+      </div>
     </li>
```

Comment:
- [ ] There is some red line on your page please remove them
![bookstore](https://user-images.githubusercontent.com/44745855/140497604-ad794af1-36c8-4654-95aa-089293c372fe.png)
- [ ] I suggest that you use the same color used on the design maybe this `#0290ff`